honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,280 @@
+    )
+
+    transform: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='Optional transform input to scale the thickness and reorient '
+                    'the up vector (default: None).'
+    )
+
+    bsdf_data: bytes = Field(
+        ...,
+        description='BSDF xml file data as bytes.'
+    )
+
+    front_diffuse_reflectance: List[float] = Field(
+        default=None,
+        description='Optional additional front diffuse reflectance as sequence of '
+                    'numbers (default: None).'
+    )
+
+    back_diffuse_reflectance: List[float] = Field(
+        default=None,
+        description='Optional additional back diffuse reflectance as sequence of '
+                    'numbers (default: None).'
+    )
+
+    diffuse_transmittance: List[float] = Field(
+        default=None,
+        description='Optional additional diffuse transmittance as sequence of '
+                    'numbers (default: None).'
+    )
+
+
+class Glow(ModifierBase):
+    """Radiance Glow material."""
+
+    type: constr(regex='^Glow$') = 'Glow'
+
+    r_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the red channel glow '
+                    '(default: 0).'
+    )
+
+    g_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the green channel glow '
+                    '(default: 0).'
+    )
+
+    b_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the blue channel glow '
+                    '(default: 0).'
+    )
+
+    max_radius: float = Field(
+        default=0,
+        description='Maximum radius for shadow testing (default: 0). '
+    )
+
+
+class Light(ModifierBase):
+    """Radiance Light material."""
+
+    type: constr(regex='^Light$') = 'Light'
+
+    r_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the red channel of the light '
+                    '(default: 0).'
+    )
+
+    g_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the green channel of the light '
+                    '(default: 0).'
+    )
+
+    b_emittance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the blue channel of the light '
+                    '(default: 0).'
+    )
+
+
+if __name__ == '__main__':
+    print(ModifierBase.schema_json(indent=2))
+    print(Void.schema_json(indent=2))
+    print(Plastic.schema_json(indent=2))
+    print(Glass.schema_json(indent=2))
+    print(BSDF.schema_json(indent=2))
+    print(Glow.schema_json(indent=2))
+    print(Light.schema_json(indent=2))
+    print(Trans.schema_json(indent=2))
```

Comment:
Why do we do this here? Is it part of the checking process or is it required? I would remove all these if it is for local checks.

Reply:
@mostaphaRoudsari  

I noticed this was implemented in the energy honeybee-schema modules, so copied it over so I could use it for informal testing. I'm not sure if there's another reason for it, but @chriswmackey can let us know, else I can get rid of it.

Reply:
I agree that we can get rid of it across all of the modules in honeybee-schema. It serves no purpose after the initial testing is done.